dist/assests/css/style.css

```diff
@@ -51,13 +52,17 @@ ul {
   display: flex;
   justify-content: space-between;
 }
+
 .main-header__nav .left {
   margin-left: 2rem;
 }
+
 .main-header__nav .right {
   margin-right: 2rem;
 }
-.main-header__nav ul, .main-header__nav ul li {
+
+.main-header__nav ul,
+.main-header__nav ul li {
```

Comment:
try to avoid element selectors in CSS whenever possible, CSS classes should be your main way of styling.
recommended reads:
https://css-tricks.com/strategies-keeping-css-specificity-low/
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

Reply:
I will cooperate with @ahmedsamir96m. to avoid it in the  global style and other sections styles as well